.github/workflows/deployment.yml

```diff
@@ -61,6 +64,7 @@ jobs:
             ./ios/flare.xcodeproj/project.pbxproj
             ./ios/flare/Info.plist
             ./version.txt
+            ./env
```

Comment:
I think I might be misunderstanding what is going on here, this says `env` while the file created above is `.env`. But we likely don’t want this file stored as an artifact, since it contains a lot of secrets?

Reply:
Thank you for this :)